.pipelines/onebranch/pipeline.image.mirror.pullrequest.yml

```diff
@@ -0,0 +1,81 @@
+#################################################################################
+#                               OneBranch Pipelines                             #
+# This pipeline was created by EasyStart from a sample located at:              #
+#   https://aka.ms/obpipelines/easystart/samples                                #
+# Documentation:  https://aka.ms/obpipelines                                    #
+# Yaml Schema:    https://aka.ms/obpipelines/yaml/schema                        #
+# Retail Tasks:   https://aka.ms/obpipelines/tasks                              #
+# Support:        https://aka.ms/onebranchsup                                   #
+#################################################################################
+
+trigger: none
+pr: none
+
+parameters:
+- name: vsoDeployerBuildID
+  type: string
+  default: latest
+
+variables:
+  CDP_DEFINITION_BUILD_COUNT: $[counter('', 0)] # needed for onebranch.pipeline.version task https://aka.ms/obpipelines/versioning
+  LinuxContainerImage: centos:centos7   # Docker image which is used to build the project https://aka.ms/obpipelines/containers
+  DEBIAN_FRONTEND: noninteractive
+  vso-deployer-pipeline-id: 206753
+  vso-project-id: 5d69ab04-7ded-49dc-84d5-bbbcac4add8d
```

Comment:
Should vs-deployer-pipeline-id and vso-project-id be hard coded?  What are they?  Same question in all the places, will not duplicate it.

Reply:
These two vars point to the Onebranch pipeline build definition that creates the ```aro``` binary and ```aro-rp``` container.  
For mirroring, regardless if it's INT, PROD, or another environment, we are just publishing build artifacts and manifests that Ev2 will consume.  Ev2 will determine the appropriate environment for the actual deployment.  This mirroring Onebranch pipeline only takes the ```aro``` binary and includes it within the Ev2 package (```ServiceGroupRoot/bin```)

The existing INT/PROD mirroring pipelines have these as exposed variables, but they contain the same values between them.  We can make these two variable user-assignable at run time, if needed, but since this is just mirroring images and not running a service, which may require a specific build, we did not think that was necessary.  However, let us know how you want to proceed and we can make the requested changes.

Reply:
For now I'm good with this. Maybe create a card to track possible future change as a touch point to check if things need adjusted.